pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -114,8 +117,18 @@ stdenv.mkDerivation ({
     ${postPatch}
   '';
 
+  # For OTP 27+ we need ex_doc to build the documentation
+  # When exdocSupport is enabled, patch ex_doc and setup necessary assets
   preConfigure = ''
     ./otp_build autoconf
+  '' + optionalString exdocSupport ''
+    mkdir -p $out/bin
+    cp ${exdoc}/bin/.ex_doc-wrapped $out/bin/ex_doc
+    sed -i "1 s:^.*$:#!$out/bin/escript:" $out/bin/ex_doc
+    export EX_DOC=$out/bin/ex_doc
```

Comment:
Point the configure script to our newly patched `ex_doc`

Note: it took me awhile to realize this. Ideally, we could pass this as `configureFlags`